src/library_webgpu.js

```diff
@@ -1659,6 +1684,10 @@ var LibraryWebGPU = {
           timestampWriteCount,
           {{{ makeGetValue('descriptor', C_STRUCTS.WGPURenderPassDescriptor.timestampWrites, '*') }}});
       }
+
+      if (maxDrawCount) {
+        desc["maxDrawCount"] = maxDrawCount;
+      }
```

Comment:
nit: this can be unconditional (just include it in `desc` instead of using `if()`)

Reply:
This seems not changed?

Reply:
Fixed. (Likely commit not upload from another machine)